### PR TITLE
Documentation: Fixing grammatical errors in the documentation #886

### PR DESCRIPTION
**Pull request by tushar-97**
Documentation: Fixing grammatical errors in the documentation #886

### CODE REVIEW

**Comment by bari12** [comment]
Hi @tushar-97 ,
The second PR according to the https://github.com/rucio/rucio/blob/master/CONTRIBUTING.rst guide is missing. Please add.

**Comment by tushar-97** [comment]
Hi @bari12 
I have added the second PR for the next branch. #905

File: doc/source/replica_workflow.rst [hunk 1]
```diff
@@ -16,11 +16,10 @@ additional information describing the file is necessary, such as meta-data, the
 
 Rucio supports pluggable algorithms for both deterministic and non-deterministic algorithms. This section explains a few of them.
 
-Deterministc algorithm based on hashes
+Deterministic algorithm based on hashes
 --------------------------------------
 
-The hash deterministic algorithm is an algorithm commonly used in Rucio. The advantage of this algorithm is that, due to the characteristics of cryptographic hash functions,
-that the files are evenly distributed to directories. This can be an important characteristic for storage systems whose access performance degrades based on the number
+The hash deterministic algorithm is an algorithm commonly used in Rucio. The advantage of this algorithm is that, due to the characteristics of cryptographic hash functions,the files are evenly distributed to directories. This can be an important characteristic for storage systems whose access performance degrades based on the number
```
Comment (bari12) [inline doc/source/replica_workflow.rst]: Space missing after comma

**Review by bari12 — CHANGES_REQUESTED** [review]

**Comment by tushar-97** [comment]
@bari12 Please review. I have made the necessary changes in both the PRs.

**Comment by bari12** [comment]
Hi @tushar-97 
It's fine now but I just realised you a separate branch for the second PR. This is really not needed. You can do the two PRs (one against master, one against next) from the same branch. `tushar-97:patch-branch`. So repeating the work on the next branch was really not needed 😄 
Please delete the other PR and re-create it with the right branch.

**Review by vingar — APPROVED** [review]

**Comment by tushar-97** [comment]
Hi @bari12 
Thanks for your patience and helping me out with this PR. I have submitted another PR #911 (for next) with the correct branch. I hope this is sufficient.

**Review by bari12 — APPROVED** [review]

**Pull request merged**